clustered/server/src/main/java/org/ehcache/clustered/server/internal/messages/EhcacheSyncMessageCodec.java

```diff
@@ -230,7 +233,11 @@ private EhcacheSyncMessage decodeMessageTracker(ByteBuffer message) {
         }
       }
     }
-    return new EhcacheMessageTrackerMessage(trackedMessages);
+    Integer segmentId = decoder.int32(MESSAGE_TRACKER_SEGMENT_FIELD);
+    if (segmentId == null) {
+      segmentId = EhcacheMessageTrackerMessage.UNKNOWN_SEGMENT;
```

Comment:
Is this temporary? So only for backward compatibility? Can you comment?

Reply:
it is a change isolated in its own commit, was planning to immediately open a PR flagged `3.5` to revert it.